stripevpaypal.md

```diff
@@ -0,0 +1,115 @@
+# Stripe and PayPal Payment Setup on Squarespace  
+
+****Authored by:****  [Ryan Lang](https://github.com/langrh), 1st yr. college    
+
+****Last Updated:**** September 21, 2023
+
+## Overview
+
+Stripe and PayPal are two payment types on Squarespace, each with their own pros and cons. However, both can be used simultaneously on your website. This guide will explain how to set up and use both payment processors, along with a comparison between them.
+
+## How to Set Up
+
+1. On Squarespace, navigate to *Commerce -> Payments*.
+2. Once there, PayPal and Stripe will show up as payment methods.
+3. For both, an account must be linked to use.
+4. To create an account for Stripe, navigate to [Stripe Registration](https://dashboard.stripe.com/register).
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.

```

Comment:
This isn't very detailed instruction. Are there at least prompts for folks to follow?

Reply:
We will have to revisit this is a later version. As we will continually update this and other documents over time